packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -1640,6 +1645,9 @@ export default function makeKernelKeeper(
             // deleted). Message delivery should use that, but not us.
             const ownerKey = `${kref}.owner`;
             let ownerVatID = kvStore.get(ownerKey);
+            if (!ownerVatID) {
+              lostKrefs.push(kref);
+            }
```

Comment:
So I was chatting with Brian, and not having a owner vatID is something that can normally happen when the exporting vat was terminated (including merely upgraded for non-durable exports I believe), but the object is still imported by other vats.

A lost kref should really be only when the kernel no longer knows about the kref at all. This might require a new call to get the state of the kref and its refcount.

Reply:
https://github.com/Agoric/agoric-sdk/pull/12145/commits/fe7b5430b248da7d0588def54d297ed151715b78